bodhi/server/consumers/automatic_updates.py

```diff
@@ -91,6 +91,10 @@ def __call__(self, message: fedora_messaging.api.Message) -> None:
             log.debug(f"Koji build info for {bnvr} doesn't contain 'owner_name'.")
             return
 
+        if kbuildinfo['owner_name'] in config.get('automatic_updates_blacklist'):
+            log.debug(f"{bnvr} owned by a user in automatic_updates_blacklist, skipping.")
```

Comment:
maybe we should also log the `owner_name`?

Reply:
done, and updated the commit! thanks!